src/App.js

```diff
@@ -1,12 +1,9 @@
-import * as React from 'react';
 import Calculator from './components/Calculator';
 
-export default class App extends React.PureComponent {
-  render() {
-    return (
-      <div className="App">
-        <Calculator />
-      </div>
-    );
-  }
-}
+const App = () => (
+  <div className="App">
+    <Calculator />
+  </div>
+);
+
+export default App;
```

Comment:
- [ ] [OPTIONAL] As requested by the previous reviewer, it is good to use the arrow function to keep javascript best practices, to avoid the linter error you were getting, you can do something like this 👇 
```
const App = () => {
  render() {
    return (
      <div className="App">
        <Calculator />
      </div>
    );
  }
}

export default App
```